book/templates/chapter-template/chapter-content/chapter-personal-stories.md

```diff
@@ -0,0 +1,55 @@
+> - A blog-style article about the subject written from their perpective - for reference, see the [personal story](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html) in the Research Data Management Chapter.
+> - A blog-style article written about the subject written from your perspective - in this case, it may be beneficial to include direct quotes from the subject in your article where possible. 
+
+## `Subject Name`: `Story Caption`
+
+> The title of the personal story should start with the name of the subject followed by a short caption that captures the overall message of the story.
+> For example, the [personal story](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html) in the Research Data Management chapter is aptly titled: `Susanna-Assunta Sansone: From FAIR Co-Author to FAIR Doer`
+
+> Regardless of how you choose to format the personal story, ensure that it contains the elements itemised in list below. 
+> Use the items to structure your content but do not include them in your contribution. 
+> Feel free to use images, external resouces, or code blocks to tell a compelling story.
+> Above all, centre the personal story around how readers are expected to benefit from the subject's experiences.
+
+> ### Key Elements of a Personal Story
+> - Introduce the subject, their role, what they are working on/have worked on, and provide links to their online activities and profiles (if the subject consents).
+>   - An introduction will help readers connect and engage with the subject better
+> - Describe how the subject is practicing your chapter's concepts in their work.
+>   - The personal story should give the reader some background about what the subject is working on
+> - Explain how your chapter's concepts has impacted the subject's work.
+>   - Impact could be positive or negative. It is important that readers get a nuanced view of how your chapter's key ideas work in a real-world context.
+> - Describe any challenges the subject faced while practicing the chapter's key ideas
+> - Communicate any thoughts or reflections the subject has about your chapter's topic.
+>   - What did the subject learn from practicing the chapter's ideas in their project? What would they have done differently? What are they most proud of?
+
+
+<!-- IMPORTANT!
+
+- Use this template to create a personal story.
+- Refrain from writing very long content as readers may be unwilling to read them. Rather, you should split long content into smaller subchapters if necessary.
+
+
+
+BEFORE YOU GO
+
+- Have a look at the Style Guide and the Maintaining Consistency chapters to ensure that you have followed the relevant recommendations on
+  - Avoiding HTML
+  - Consecutive headers
+  - Labels and cross referencing
+  - Using images
+  - Latin abbreviations
+  - References and citations
+  - Title casing
+  - Matching headers with reference in table of content
+
+-->
```

Comment:
```suggestion

## BEFORE YOU GO

> - Use this template to create a personal story.
> - Refrain from writing very long content as readers may be unwilling to read them. Rather, you should split long content into smaller subchapters if necessary.
> - Have a look at the Style Guide and the Maintaining Consistency chapters to ensure that you have followed the relevant recommendations on
>   - Avoiding HTML
>   - Consecutive headers
>   - Labels and cross referencing
>   - Using images
>   - Latin abbreviations
>   - References and citations
>   - Title casing
>   - Matching headers with reference in table of content
```

Reply:
I think it's ok to have these visible in the rendered file. A disclaimer can be added on the top that asks authors to remove the quoted block.

Reply:
In consistency with the chapter template, I am adding this change.